katello/hooks/boot/30-upgrade.rb

```diff
@@ -4,9 +4,3 @@
   "Run the steps necessary for an upgrade such as migrations, rake tasks, etc.",
```

Comment:
This line can use an update to state it's just there to warn users it's no longer needed.